README.md

```diff
@@ -14,8 +14,15 @@ mv data-100m-all.txt ../datasets/.
 
 ```
 
+For the addition grokking run:
+
+```bash
+ cargo run --release -- --min 4 --max 5 --size 7000 --seed 32 --save-file-path data-grokking-addition.txt --operators +
+mv data-grokking-addition.txt ../datasets/.
```

Comment:
I think the instruction to build them should be enough, no need to commit the dataset. Since the seed is fixed it is reproducible